docs/contracts.rst

```diff
@@ -129,7 +129,7 @@ Each Contract Factory exposes the following methods.
 
     Returns the transaction hash for the deploy transaction.
 
-.. py:classmethod:: Contract.eventFilter(event_name, filter_params=None)
```

Comment:
Given that this method is still present, we should leave it in the documentation and just mark it as having been deprecated.

```python
.. warning:: The `Contract.eventFilter` method has been deprecated in favor of .......
```